### PR TITLE
docs(nuget): rewrite readme 

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Rewrite the Nuget readme

## Context

Missed this document when working on PR #15846.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/manager/nuget/readme.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
+For Renovate to work with .NET Framework projects, you need to update these files so they match the new SDK-style format:
+
+- `.csproj`
+- `.fsproj`
+- `.vbproj`
+- `.props`
+- `.targets`
 
 The NuGet manager also supports `global.json` and `dotnet-tools.json` SDK files.
```
Comment (HonkingGoose) [inline lib/modules/manager/nuget/readme.md:21]: Do we support `global.json` and `dotnet-tools.json` anymore?

File: lib/modules/manager/nuget/readme.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
-The `nuget` configuration object is used to control settings for the NuGet package manager.
-The NuGet package manager supports a SDK-style `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format, as described [here](https://natemcmaster.com/blog/2017/03/09/vs2015-to-vs2017-upgrade/).
```
Comment (HonkingGoose) [inline lib/modules/manager/nuget/readme.md:2]: I've removed the link from the docs, but maybe we still want it? I'd prefer to link to some _official_ docs instead.

Comment (rarkins) [reply lib/modules/manager/nuget/readme.md:2]: Let's remove

File: lib/modules/manager/nuget/readme.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
-This means that .NET Core projects are all supported but any .NET Framework projects need to be updated to the new `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format in order to be detected and supported by Renovate.
+You can use the `nuget` configuration object to control the settings for the NuGet package manager.
```
Comment (viceice) [inline lib/modules/manager/nuget/readme.md:1]: The `nuget` config object is also used by `nuget` datasource. We should probably remove this line and instead suggest to use package rules.

File: lib/modules/manager/nuget/readme.md (hunk 1 above)
Comment (viceice) [reply lib/modules/manager/nuget/readme.md:21]: yes we do support them

https://github.com/renovatebot/renovate/blob/dca3418bbdbf1024ac6807e591562b4042bc846d/lib/modules/manager/nuget/index.ts#L10-L15

File: lib/modules/manager/nuget/readme.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
+
+The NuGet package manager supports these SDK-style files and formats:
+
+- `.csproj`
+- `.fsproj`
+- `.vbproj`
+- `.props`
+- `.targets`
```
Comment (HonkingGoose) [inline lib/modules/manager/nuget/readme.md:9]:
```suggestion
- `.targets`
- `global.json`
- `dotnet-tools.json`
```

File: lib/modules/manager/nuget/readme.md (hunk 1 above)
Comment (HonkingGoose) [inline lib/modules/manager/nuget/readme.md:21]:
```suggestion
```

**Review by HonkingGoose — COMMENTED** [review]
Should we put `global.json` and `dotnet-tools.json` in the list?

File: lib/modules/manager/nuget/readme.md (hunk 3 above)
Comment (viceice) [reply lib/modules/manager/nuget/readme.md:1]: ☝️

**Review by viceice — COMMENTED** [review]
Still an open question

File: lib/modules/manager/nuget/readme.md (hunk 3 above)
Comment (HonkingGoose) [reply lib/modules/manager/nuget/readme.md:1]:
```suggestion
Use [`packageRules`](https://docs.renovatebot.com/configuration-options/#packagerules) to control the behavior of the NuGet package manager.
```

How about something like this?

**Review by viceice — DISMISSED** [review]

**Pull request merged**